### ISSUE TITLE
No longer that free

### ISSUE THREAD

**Issue by crocodile2u** [benmajor/ExchangeRatesAPI#8]
Apparently, "the awesome, free ExchangeRatesAPI" is no longer that free. As of today, our services relying upon this API started to fail, and I discovered that exchangeratesapi.io started to require API key, and only 250 requests/month are free. Everyone needs to run their server.

**Comment by benmajor** [comment]
Okay, thanks for the heads-up, this is a massive shame as it was an excellent resource. It looks as though this is also going to require a rewrite for the SDK!

**Comment by crocodile2u** [comment]
> Okay, thanks for the heads-up, this is a massive shame as it was an excellent resource. It looks as though this is also going to require a rewrite for the SDK!

Exactly! I actually came across your SDK just now, I used my own very simplistic client in our company's private packagist. Already fixed it there, it only requires an extra GET parameter: access_key=API_KEY

**Comment by crocodile2u** [comment]
@benmajor , you might want to take a look here: ratesexchange.eu

They claim their API to be free of charge and unlimited, although it still  requires an API key. By the looks of it, the schema is compatible with our good friend ExchangeRatesAPI. Cheers.

**Comment by aglipanci** [comment]
Same here: https://exchangerate.host

Their API is also 100% compatible with ExchangeRatesAPI.

**Comment by benmajor** [comment]
Thanks @aglipanci - that looks like it could be a great drop-in replacement. I'll take a look.

**Comment by Schyzophrenic** [comment]
I confirm that replacing the apiUrl with https://api.exchangerate.host/ works very well for my use cases. 
I actually started by adding the key required by the previous API, but I was going from limitation to limitation so I moved on. 
It seems it is not only the number of calls which is limited, which would have been workable, but more features are available to the higher tiers. 
It started with the SSL mode, then the next one was the symbols param. This is when I stopped investigating .
